bitrise.yml

```diff
@@ -78,6 +79,53 @@ workflows:
         inputs:
         - dir_to_check: $CLONE_INTO_DIR
 
+  error_test:
```

Comment:
I do not see the added value of this test, what we changed here is commands are returning the output in case of exist status error